ncmcm/visualisers/latent_space.py

```diff
@@ -123,11 +123,9 @@ def _plot_ps(self, fig, ax, colors=None, **kwargs):
 
         for i in range(len(self.y) - 1):
             d = (self.y[i + 1] - self.y[i])
-            kwargs.setdefault('arrow_length_ratio', 0.1 / np.linalg.norm(d))
-            kwargs.setdefault('linewidths', 1)
             ax.quiver(self.y[i, 0], self.y[i, 1], self.y[i, 2],
                       d[0], d[1], d[2],
-                      color=color_dict[self.b[i]], **kwargs)
+                      color=color_dict[self.b[i]], arrow_length_ratio=0.1/np.linalg.norm(d), linewidths=1, **kwargs)
```

Comment:
Doesn't this effectively lead to the same output? Instead of setting `arrow_length_ratio` in             `ax.quiver`, it is set in `kwargs.setdefault('arrow_length_ratio', 0.1 / np.linalg.norm(d))`. The latter version is more flexible since it allows the user to change the arrow size according the what suits the plot best.

Reply:
On my end it didn't lead to the same output, when ``arrow_length_ratio`` is set directly in ``ax.quiver`` it's recalculated at each iteration. In the for loop, I got all the values of ``0.1/np.linalg.norm(d)`` and they ranged from 0.2 to 42

While if it is set in ``kwargs.setdefault`` it just has one value for all iterations. And at least on my end, the results seem very messy when using just one value.

Some examples:
kwargs.setdefault('arrow_length_ratio', 0.1 / np.linalg.norm(d))
![phase_space_setdefault](https://github.com/user-attachments/assets/8679dc94-3d47-4c38-8aa2-e4bbd09ff76c)

arrow_length_ratio = 5
![phase_space_5](https://github.com/user-attachments/assets/2cc4851b-a83b-4366-bf13-89a6d3061703)


arrow_length_ratio = 10
![phase_space_10](https://github.com/user-attachments/assets/8aa9d856-b8a8-466b-a1a0-bafa1df3e978)

Reply:
On my end it didn't lead to the same output, when ``arrow_length_ratio`` is set directly in ``ax.quiver`` it's recalculated at each iteration. In the for loop, I got all the values of ``0.1/np.linalg.norm(d)`` and they ranged from 0.2 to 42

While if it is set in ``kwargs.setdefault`` it just has one value for all iterations. And at least on my end, the results seem very messy when using just one value.

Some examples:
kwargs.setdefault('arrow_length_ratio', 0.1 / np.linalg.norm(d))
![phase_space_setdefault](https://github.com/user-attachments/assets/8679dc94-3d47-4c38-8aa2-e4bbd09ff76c)

arrow_length_ratio = 5
![phase_space_5](https://github.com/user-attachments/assets/2cc4851b-a83b-4366-bf13-89a6d3061703)


arrow_length_ratio = 10
![phase_space_10](https://github.com/user-attachments/assets/8aa9d856-b8a8-466b-a1a0-bafa1df3e978)

Reply:
That makes sense, thank you.

I would like to have a way by which one can control the size of arrows, and ideally other parameters of the quiver plot, directly through the plot_phase_space method, without having to go back to the src to change code. For example, in some plots I would like to have very small arrowheads, and in others larger ones, and in yet other, none at all.